.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Purpose
+
+(FILL ME IN) This section describes why this PR is here. Usually it would include a reference 
+to the tracking task that it is part or all of the solution for.
+
+### Declarations
+
+Check these if you believe they are true
+
+- [ ] The codebase is in a better state after this PR
+- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
+- [ ] The level of testing this PR includes is appropriate
+- [ ] User facing strings, if any, are extracted into `*.resx` files
+- [ ] All tests pass using the self-service CI.
+- [ ] Snapshot of UI changes, if any.
+- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
```

Comment:
Some of these points do not yet apply to this project"
- Resources: We are not using them, mainly because we don't generate any messages here. This is just a thin wrapper around the Package Manager REST API.
- Tests: Right now we don't have any, although we definitely should have some.

You may choose to remove them just to avoid confusion.

Reply:
Will remove

Reply:
Sorry, just saw this one, which also does not apply `- [ ] Snapshot of UI changes, if any.`

Reply:
Thanks thats true